### PR TITLE
Feature/AT-9033 (tweaks)

### PR DESCRIPTION
**Pull request by jeffsegal-ccpo**
A few changes that didn't make the cut before the original feature branch merged.

### CODE REVIEW

File: document-generation/utils/sampleTestData.ts [hunk 1]
```diff
@@ -1601,9 +1603,9 @@ export const sampleJustificationAndApproval = {
     },
     scope: "J & A Scope Test 1",
     contractingShop: "DITCO",
-    purchaseRequestNumber: "OYYMM-000-000-000000, ",
-    agencyLabel: "Business Transformation Agency (BTA)",
-    taskOrderType: "Firm Fixed Price",
+    purchaseRequestNumber: "OYYMM-000-000-000000",
+    agencyLabel: "Business Transformation Agency (BTA)/Jeff Org J&A",
+    taskOrderType: "firm-fixed-price",
```
Comment (jasonburkert-ccpo) [inline document-generation/utils/sampleTestData.ts:1608]: I find this surprising. Why is this value kebab case?

File: document-generation/utils/sampleTestData.ts [hunk 2]
```diff
@@ -1552,6 +1552,7 @@ export const sampleRequirementsChecklistRequest = {
 export const sampleJustificationAndApproval = {
   documentType: "JUSTIFICATION_AND_APPROVAL",
   templatePayload: {
+    estimatedValueFormatted: "$50,000,000.00",
```
Comment (jasonburkert-ccpo) [inline document-generation/utils/sampleTestData.ts:1555]: Good addition

File: document-generation/utils/sampleTestData.ts [hunk 3]
```diff
@@ -1585,10 +1586,11 @@ export const sampleJustificationAndApproval = {
       title: null,
     },
     otherContractingShopFullAddress: null,
+    appropriationFundsType: "Research, Development, Test & Evaluation",
     title: "Jeff J&A Test 1",
     jwccContractNumber: "HC105023D0002",
-    estimatedValue: "$0.00",
-    organizationFullAddress: "Jeff Org J&A\n123 Fake St\nSpringfield, OH 12345\nUnited States of America",
+    estimatedValue: 50000000.0,
```
Comment (jasonburkert-ccpo) [inline document-generation/utils/sampleTestData.ts:1592]: 👍

**Review by jasonburkert-ccpo — APPROVED** [review]

File: document-generation/utils/sampleTestData.ts (hunk 1 above)
Comment (jeffsegal-ccpo) [reply document-generation/utils/sampleTestData.ts:1608]: DITCO preference

**Pull request merged**